scrapper.py

```diff
@@ -18,6 +25,34 @@
 dictwriter = csv.DictWriter(csv_file, fieldnames=fieldnames)
 dictwriter.writeheader()
 
+
+#Selenium scrapper
+def get_q(pages, tag):
+    URL = 'https://stackoverflow.com/questions'
+    display_dict = {}
+    while_counter = 1
+    while while_counter < int(pages):
+        DRIVER.get(URL)
```

Comment:
this is probably a better place to add a try catch, essentially when ever you make a request